README.md

```diff
@@ -48,7 +48,6 @@ pre-configured.
   Create Cognito User Pool Clients.
 
 - *Features not yet implemented*:
-  [`cognito_user_pool_domain`](https://www.terraform.io/docs/providers/aws/r/cognito_user_pool_domain.html)
```

Comment:
need to be moved to implemented not just deleted ;)

Reply:
resolved in 28d556a